partition_tables_templates/kickstart_uefi_custom.erb

```diff
@@ -0,0 +1,35 @@
+<%#
+kind: ptable
+name: Kickstart UEFI custom
+model: Ptable
+oses:
+- CentOS
+- Fedora
+- RedHat
+%>
+zerombr
+clearpart --all --initlabel
+<% if @host.pxe_loader.include?("UEFI") -%>
+  
+part /boot/efi --fstype="efi" --ondisk=sda --size=200 --fsoptions="umask=0077,shortname=winnt"
+part /boot --fstype="ext4" --ondisk=sda --size=1024
+part pv_sda --fstype="lvmpv" --ondisk=sda --size=1024 --grow
+volgroup vg_sda --pesize=4096 pv_sda
+logvol swap --fstype="swap" --size=4096 --name=lv_swap --vgname=vg_sda
+logvol / --fstype="ext4" --percent=40 --name=lv_root --vgname=vg_sda
+#logvol / --fstype="ext4" --grow --name=lv_root --vgname=vg_sda
+
+<% elsif @host.pxe_loader.include?("BIOS") -%>
+
+part /boot --fstype="ext4" --ondisk=sda --size=1024
+part pv_sda --fstype="lvmpv" --ondisk=sda --size=1024 --grow
+volgroup vg_sda --pesize=4096 pv_sda
+logvol swap --fstype="swap" --size=4096 --name=lv_swap --vgname=vg_sda
+logvol / --fstype="ext4" --percent=40 --name=lv_root --vgname=vg_sda
+#logvol / --fstype="ext4" --grow --name=lv_root --vgname=vg_sda
+
+<% else -%>
+  
+autopart <%= host_param('lv_root_percent') %>
```

Comment:
https://docs.fedoraproject.org/en-US/fedora/rawhide/install-guide/appendixes/Kickstart_Syntax_Reference/#sect-kickstart-commands-autopart

This doesn't look like it accepts a percentage. Am I missing something?